src/config/server.js

```diff
@@ -34,7 +34,7 @@ export const serverConfig = {
     port: {
       doc: 'The port to bind.',
       format: 'port',
-      default: 3002,
+      default: 3000,
```

Comment:
I think this port change should be done as its own PR so we can test this in the dev and if there are any issues we can fix foward or roll back

Reply:
The port needs to be `3000` or it will be not be possible to authenticate with Defra ID which is the purpose of this ticket. (See redirect comment)

So I'd do that change and update the documentation as part of this PR otherwise you won't be able to run this locally if you merge it still as `3002`